span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/SpanFilter.java

```diff
@@ -26,20 +24,16 @@
    * ["messaging.destination_kind:queue,messaging.operation:receive,messaging.system:jms"] drops all
    * spans which have all 3 attribute:value pairs.
    */
-  private static final String SPAN_DROP_CRITERION_CONFIG = "processor.spanDropCriterion";
+  public static final String SPAN_DROP_CRITERION_CONFIG = "processor.spanDropCriterion";
```

Comment:
Can't `processor.rootSpanDropCriterion` be covered in `processor.spanDropCriterion` itself?